tigerpath/majors_and_certificates/majors/ANT_2018.json

```diff
@@ -120,54 +151,137 @@
               "name": "Required Courses",

```

Comment:
The required courses for this track [seem to have changed](https://anthropology.princeton.edu/undergraduate/tracks/medical-anthropology) over the last few days.